stdeb/util.py

```diff
@@ -93,10 +93,9 @@ def check_call(*popenargs, **kwargs):
     ('guess-conflicts-provides-replaces=',None,
      'If True, attempt to guess Conflicts/Provides/Replaces in debian/control '
      'based on apt-cache output. (Default=False).'),
-    ('with-python2=',None,
-     help_str_py2),
-    ('with-python3=',None,
-     help_str_py3),
+    ('with-python-virtualenv=',None, """trick for using dh_virtualenv"""),
```

Comment:
This help text should probaby be handled in the similar way to the other python versions.